corehq/apps/accounting/interface.py

```diff
@@ -930,9 +937,9 @@ def rows(self):
                     text=record.date_created.strftime(USER_DATE_FORMAT),
                     sort_key=record.date_created.isoformat(),
                 ),
-                record.payment_method.account.name,
-                record.payment_method.billing_admin.domain,
-                record.payment_method.billing_admin.web_user,
+                self.account(record).name,
```

Comment:
(trivial) I prefer ```get_account``` or something that establishes the relationship between the argument and the return value